scripts/generator/graphql_generator/csharp/SDK/Android/AndroidPayCheckoutResponse.cs.erb

```diff
@@ -0,0 +1,41 @@
+#if UNITY_ANDROID
+namespace <%= namespace %>.SDK.Android {
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+
+    public class AndroidPayCheckoutResponse : Serializable {
```

Comment:
This name sucks, but I couldn't think of anything better. This type is just a wrapper for the updated cart information that will be transferred from Unity back to the Android library. I could have gone with just plain JSON strings and dictionaries, but using a model class seems easier and type-safe, since all JSON fields are explicitly declared. I tried to derive the name from `ApplePayEventResponse`, which seems to kinda have the same purpose. Does anything along the lines of `UpdatedAndroidPayCart` work? I'm totally open for suggestions. Also, I plan to add XML documentation for all these classes and methods I've created, this will be helpful to understand the purpose of any class and method.

Reply:
We could follow the Apple Pay convention and name it `AndroidPayEventResponse`? Not sure if that works or not.

Reply:
https://github.com/Shopify/unity-buy-sdk/pull/321/commits/0f01a23043c1cc088a555d3ceea7d0b46c7e854e

Reply:
https://github.com/Shopify/unity-buy-sdk/pull/321/commits/d8e303d5ac5a2bc29ab3844956664f94187ab21e